### PR TITLE
feat: parse Label element

### PR DESCRIPTION
**Pull request by RomeroDiver**
This is a possible fix for https://github.com/videojs/video.js/issues/6960

### CODE REVIEW

File: src/inheritAttributes.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
+import decodeB64ToUint8Array from '@videojs/vhs-utils/dist/decode-b64-to-uint8-array';
+import resolveUrl from '@videojs/vhs-utils/dist/resolve-url';
 import window from 'global/window';
+import errors from './errors';
+import { parseAttributes } from './parseAttributes';
```
Comment (RomeroDiver) [inline src/inheritAttributes.js:5]: The auto-import plugin, will remove this now

File: test/toM3u8.test.js [hunk 2]
```diff
@@ -701,3 +701,48 @@ QUnit.test('dynamic playlists with suggestedPresentationDelay', function(assert)
...
+      duration: 0,
+      bandwidth: 10000,
+      periodIndex: 1,
+      mimeType: 'video/mp4'
+    },
+    segments: []
+  }];
+  const output = toM3u8(input);
+
+  assert.ok(label in output.mediaGroups.AUDIO.audio);
```
Comment (gkatsev) [inline test/toM3u8.test.js:747]:
```suggestion
  assert.ok(label in output.mediaGroups.AUDIO.audio, 'a label exists');
  assert.equal(output.mediaGroups.AUDIO.audio.label, label, 'we got the label set in the mpd');
```

**Comment by gkatsev** [comment]
Thanks for PR, just one small addition to the test, code looks good!

File: test/toM3u8.test.js (hunk 2 above)
Comment (RomeroDiver) [reply test/toM3u8.test.js:747]: I'm not sure this is the correct assertion, the second one. There is no property label on audio object.
The structure of the audio object is similar to 
```javascript
{
  audio: {
    [label]: {...}
  }
}
```

**Comment by slasktrat** [comment]
+1 for this!

**Review by brandonocasey — APPROVED** [review]

**Pull request merged**